src/modules/dashboard/components/Dialogs/EditExpenditureDialog/ChangedBatch/ChangedBatch.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { isEmpty } from 'lodash';
+import { nanoid } from 'nanoid';
+
+import Button from '~core/Button';
+import { Colony, LoggedInUser } from '~data/index';
+import { ValuesType } from '~pages/ExpenditurePage/types';
+import { Staged } from '~dashboard/ExpenditurePage/Staged/types';
+import { Batch } from '~dashboard/ExpenditurePage/Batch/types';
+import { FormSection } from '~core/Fields';
+
+import ChangeItem from '../ChangedMultiple/ChangeItem';
+
+import styles from './ChangedBatch.css';
+
+export const MSG = defineMessages({
+  discard: {
+    id: 'dashboard.EditExpenditureDialog.ChangedBatch.discard',
+    defaultMessage: 'Discard',
+  },
+  change: {
+    id: 'dashboard.EditExpenditureDialog.ChangedBatch.change',
+    defaultMessage: 'Change Batch',
+  },
+  from: {
+    id: 'dashboard.EditExpenditureDialog.ChangedBatch.from',
+    defaultMessage: 'From',
+  },
+  changeTo: {
+    id: 'dashboard.EditExpenditureDialog.ChangedBatch.changeTo',
+    defaultMessage: 'Change to',
+  },
+  removed: {
+    id: 'dashboard.EditExpenditureDialog.ChangedBatch.removed',
+    defaultMessage: 'Recipient has been deleted',
+  },
+});
+
+const displayName = 'dashboard.EditExpenditureDialog.ChangedBatch';
+const skip = ['id', 'dataCSVUploader', 'data'];
+
+export type NewValueType = {
+  id: string;
+  key: string;
+  value?:
+    | ValuesType['recipients']
+    | string
+    | Staged
+    | Pick<
+        LoggedInUser,
+        'walletAddress' | 'balance' | 'username' | 'ethereal' | 'networkId'
+      >
+    | Batch;
+};
+
+interface Props {
+  newValues?: NewValueType;
+  colony: Colony;
+  oldValues: ValuesType;
+  discardChange: (name: string) => void;
+}
+
+const ChangedBatch = ({
+  newValues,
+  oldValues,
+  colony,
+  discardChange,
+}: Props) => {
+  const batchChanges = useMemo(
+    () =>
+      typeof newValues?.value === 'object'
+        ? Object.entries(newValues.value)
+            .filter(([key]) => {
+              return !skip.includes(key);
```

Comment:
It's a nitpick but I've noticed it in several places already. Can you please not use the `return` version of the arrow function, when there is nothing in the function other than what you return. It's a bit shorter and in my opinion cleaner to omit it.